android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -45,7 +45,8 @@
 public class WelcomeOnboardingActivity extends FirstRunActivityBase {
     // mInitializeViewsDone and mInvokePostWorkAtInitializeViews are accessed
     // from the same thread, so no need to use extra locks
-    private static final String P3A_URL = "https://brave.com/privacy/browser/#how-we-improve-brave";
+    private static final String P3A_URL =
```

Comment:
In ticket it's mentioned URL : https://support.brave.com/hc/en-us/articles/9140465918093-What-is-P3A-in-Brave-
Here missing one dash(-) at the end.

Reply:
@sujitacharya2005 it works well without `-` at the end.